src/engraving/libmscore/beam.cpp

```diff
@@ -741,36 +741,40 @@ void Beam::createBeamSegments(std::vector<ChordRest*> chordRests)
         levelHasBeam = false;
         Chord* startChord = nullptr;
         Chord* endChord = nullptr;
+        bool breakBeam = false;
 
         for (uint i = 0; i < chordRests.size(); i++) {
             ChordRest* chordRest = chordRests[i];
             if (!chordRest->isChord()) {
                 continue;
             }
             Chord* chord = toChord(chordRest);
-            if (level < chord->beams()) {
+            Beam::Mode beamMode = chord->beamMode();
+            breakBeam = level < chord->beams()
+                        && ((level >= 1 && beamMode == Beam::Mode::BEGIN32) || (level >= 2 && beamMode == Beam::Mode::BEGIN64));
+            if (level < chord->beams() && !breakBeam) {
```

Comment:
I think you'll need to change `Beam::Mode` to `BeamMode`, because it was renamed recently.